src/main/java/kitchenpos/tablegroup/dto/TableGroupRequest.java

```diff
@@ -0,0 +1,38 @@
+package kitchenpos.tablegroup.dto;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class TableGroupRequest {
+    private List<Ids> orderTables = new ArrayList<>();
+
+    public TableGroupRequest() {}
+
+    public TableGroupRequest(List<Long> ids) {
+        orderTables = ids.stream().map(Ids::new).collect(Collectors.toList());
+    }
+
+    public List<Long> ids() {
+        return orderTables.stream().map(Ids::getId).collect(Collectors.toList());
+    }
+
+    public List<Ids> getOrderTables() {
+        return orderTables;
+    }
+}
+
+class Ids{
+    private Long id;
+
+    public Ids() {
+    }
+
+    public Ids(Long id) {
+        this.id = id;
+    }
+
+    public Long getId() {
+        return id;
+    }
+}
```

Comment:
```suggestion
    }
    
    public static class Ids{
        private Long id;
    
        public Ids() {
        }
    
        public Ids(Long id) {
            this.id = id;
        }
    
        public Long getId() {
            return id;
        }
    }
}
```
이렇게 명시적으로 class 내부로 넣어주는게 좋지 않을까요?

Reply:
리뷰어님! 혹시 이 설계는 어떠한 장점이 있을까요?!

Reply:
말 그대로 TableGroupRequest의 하위 클래스라고 명시하는거죠! 구조적으로 이해하기가 더 쉽지 않을까요? 사실 무척 사소한 부분이라서 크게 신경쓰지 않아도 되리라 생각합니다 ㅎㅎ